packages/babel-plugin-transform-es2015-modules-amd/test/fixtures/amd/get-module-name-option/expected.js

```diff
@@ -1 +1,3 @@
-define("my custom module name", [], function () {});
+define("my custom module name", [], function () {
+  "use strict";
```

Comment:
Sounds like we just have to do `if (node.body.length === 0) return;` in https://github.com/babel/babel/blob/e06faa99da569d21d47dcc0c2b2a515295f94311/packages/babel-plugin-transform-strict-mode/src/index.js#L10